assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/utils/calculateBasePrice.ts

```diff
@@ -14,7 +14,7 @@ const calculateBasePrice = (state: DataState): DataState['prices'] => {
 	}
 
 	const allPrices = state?.prices;
-	if (!allPrices.length) {
+	if (!allPrices?.length) {
```

Comment:
interesting, didn't knew this one:

![image](https://user-images.githubusercontent.com/1477453/81714235-85840280-947f-11ea-86a8-fd320a0aba8c.png)